uber/templates/regform.html

```diff
@@ -621,23 +555,5 @@
         });
     {% endif %}
 
-    $(function () {
```

Comment:
Whoops. This was causing weird errors, was removed only for testing. Let me add that back.